presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java

```diff
@@ -296,4 +370,29 @@ private long calculateTotalRows(int scaleBase, double scaleFactor)
         }
         return (long) totalRows;
     }
+
+    public static Type getNumericType(TypeSignature numericTypeSignature)
+    {
+        if (numericTypeSignature.getBase().equals(DECIMAL)) {
```

Comment:
Sure, we can remove remove dynamic decimal support from TPCH connector. We can also remove this commit from this PR and add it later on. Both are fine for me.